data/mods/Xedra_Evolved/items/book_lore.json

```diff
@@ -49,5 +49,77 @@
+    "weight": "3 g",
+    "volume": "5 ml",
+    "price": 0,
+    "material": [ "paper" ],
+    "symbol": "/",
+    "color": "light_gray"
+  },
+  {
+    "id": "dollkin_tablet",
+    "type": "BOOK",
+    "name": { "str": "dollkin tablet" },
+    "description": "My dearest child, you are awaking into a cold dying world alone and without parents.  Honestly, this is for the best, for I was brought to life amongst the mortals of this realm and the parenting I would be able to pass on to you is entirely tragic.  I will spare you from that fate.  However, I have summoned some lesser creatures of the underhill realms to guard this place and you for as long as they can.  Most everything else in this world will attempt to kill you.   I have included instructions for how to make a healing item as well as a developmental potion that you can take to bring yourself further into your destiny and will grant new abilities, somewhat randomly I'm afraid.  I hope that you survive long enough to become what I hoped for you to be and that you are able to find one of the trodden paths from this place into the borderlands and from there deeper into Fae where you will hopefully find me.  All of my hopes and dreams ride with you.",
+    "weight": "3 g",
+    "volume": "5 ml",
+    "price": 0,
+    "material": [ "stone" ],
+    "symbol": "/",
+    "color": "light_gray"
+  },
+  {
+    "id": "flametouched_parchment",
+    "type": "BOOK",
+    "name": { "str": "flametouched  parchment" },
+    "description": "If you are reading this you have woken up to sentience as I planned.  My timing however was not great in that I've been forced to leave before showing you the hunting ground that I've brought you into existence in.  How I envy the experience you will have rampaging through the ashes of the old world.  There is significant danger in this world and in time you will be one of the most dangerous beings in it.  However, since that time is not yet I have summoned some lesser creatures of the underhill realms to guard this place and you for as long as they can.  Most everything else in this world will attempt to kill you.   I have included instructions for how to make a healing item as well as a developmental potion that you can take to bring yourself further into your destiny and will grant new abilities, somewhat randomly I'm afraid.  I hope that you rise to the top of this crush of predators I've been forced to abandon you among.  Good luck and good hunting.",
```

Comment:
```suggestion
    "description": "If you are reading this you have woken up to sentience as I planned.  My timing however was not great in that I've been forced to leave before showing you the hunting ground that I've brought you into existence in.  How I envy the experience you will have rampaging through the ashes of the old world.  There is significant danger in this world and in time you will be one of the most dangerous beings in it.  However, since that time is not yet I have summoned some lesser creatures of the underhill realms to guard this place and you for as long as they can.  Most everything else in this world will attempt to kill you.  I have included instructions for how to make a healing item as well as a developmental potion that you can take to bring yourself further into your destiny and will grant new abilities, somewhat randomly I'm afraid.  I hope that you rise to the top of this crush of predators I've been forced to abandon you among.  Good luck and good hunting.",
```